docs/contributing.md

```diff
@@ -0,0 +1,122 @@
+# Contributing
```

Comment:
I wonder if this content should actually be in the `CONTRIBUTING.md` file, and then embedded in the live site via the markdown embedding thing? That way the contributing doc is nice and reasonably self-contained, but the site still also contains the important information.

Reply:
Yeah, I'd tried that initially. Unfortunately getting the links to play nicely (without resorting to linking out to the hosted version) was rather complicated.